using_redcap_api/using_redcap_api.md

```diff
@@ -0,0 +1,436 @@
+
+## R and the REDCap API
+
+REDCap assumes you're going to use an R script and not an R Markdown document, but we advocate using R Markdown for just about everything you do in R, so we're going to teach that approach.
+
+<div class = "important">
+There are useful R packages that make importing REDCap data into R simpler than the code REDCap suggests.  Nevertheless, we think it's important to show you how to use the code REDCap provides, because it is reliable and reflects the current state of the REDCap API.  
+
+The R packages that streamline this effort (we like REDCapR and redcapAPI) are great and seem very stable, and we provide links to them toward the end of this module, but R packages depend on volunteers who keep them updated.  Therefore, we think it's useful to know how to use the basic method we'll share here, even if in the future you prefer a different approach.
+</div>
+
+First, open RStudio and create a new R Markdown (File > New File > R Markdown).  Give your R Markdown the title and author information you want, leave the default output choice (HTML), and click on "OK".
+
+![The new R Markdown dialog box from RStudio](media/new_r_markdown.png)<!-- style = "border :1px solid rgb(var(--color-highlight));" -->
+
+Remove all the "boilerplate" text after the setup chunk (so, delete all the lines after around line 10) and add a new empty chunk:
+
+![R Markdown document with an empty chunk](media/empty_chunk.png)<!-- style = "border :1px solid rgb(var(--color-highlight));" -->
+
+Then:
+
+* Copy the R code from the API Playground and paste it into the empty chunk
+* Remove the first line (the one that starts `#!/usr/bin`).
+* Run that chunk!
+
+After running that chunk, you should have some new objects in your "Environment".  Of particular interest is the object called `result`.  See below what we see when we run the chunk of code to get records from the cervical cancer database:
+
+![RStudio showing code on the left and environment variables including the "result" dataframe listed on the right](media/rstudio_environment.png)<!-- style = "border :1px solid rgb(var(--color-highlight));" -->
+
+Click on "result" in the environment pane to open a file viewer.  
+
+![The "result" data frame shown with its rows and columns](media/rstudio_viewer.png)<!-- style = "border :1px solid rgb(var(--color-highlight));" -->
+
+Now, your data is in a data frame and you're ready to do all sorts of work with that data (like cleaning question marks out of the data, getting summary statistics, finding correlations, visualizing data, and modeling data).  Importantly, you can re-run the chunks in this R Markdown at any time to get the latest version of the data from REDCap!  
+
+## Recap
```

Comment:
Is this section necessary? It matches the learning objectives which are then repeated on the Feedback page.

Reply:
Don't we have Recap as a requirement?  Hmm I better check!

Reply:
Nope, I was wrong, we don't have to have a Recap and I agree that in this case it's silly.  Removed.